styles.css

```diff
@@ -354,18 +359,140 @@ textarea {
   box-shadow: 1px 1px 1px #999;
 }
 
+.divider-desktop {
+  display: none;
+}
+
 /* Desktop */
 
 @media all and (min-width: 60em) {
```

Comment:
- [ ] The value for `min-width` property should be in range of `992px` - `1100px`. 60em is equal to 960px